site/static/examples/jobs/sample-job.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: batch/v1
 kind: Job
 metadata:
-  generateName: sample-job-
+  name: sample-job-
```

Comment:
Why is this a "fix"? `generateName` is a valid way of generating job names to avoid collisions. Do we have some docs that require deterministic job maybe?

Reply:
I used this yaml apply jobs and found an error, so I made changes.
```bash
➜  jobs git:(main) ✗ kubectl apply -f sample-job.yaml                                   
error: from sample-job-: cannot use generate name with apply

```

Reply:
I thought it was a typo

Reply:
This is not a typo.
You can create a Job with `kubectl create -f sample-job.yaml`.
/close

Reply:
thanks！